### ISSUE TITLE
Console Virtual Terminal Sequences

### ISSUE THREAD

**Issue by spudwa** [snakefoot/snaketail-net#147]
I currently log to a terminal console and highlight/color text using the newly available VT sequences. Could snaketail support this?

https://docs.microsoft.com/en-us/windows/console/console-virtual-terminal-sequences

**Comment by snakefoot** [comment]
So you redirect the console output to a file, and when SnakeTail is used to view this file, then SnakeTail should automatically perform coloring based on the VT-instructions?

Right now SnakeTail doesn't recognize VT-sequences.

**Comment by spudwa** [comment]
Yes I would use the C freopen function to redirect all my printf's to a file which will be tailed by snaketail. These VT sequences are the new way of doing terminal stuff as they will work for Windows and Linux/Mac OSes. I've downloaded the snaketail source code so if you could point me to the approximate place I'll have a go myself
